CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [[1.1.4](https://github.com/PerimeterX/marshmallow/compare/v1.1.3...v1.1.4)] - 2022-11-10
```

Comment:
```suggestion
## [1.1.4](https://github.com/PerimeterX/marshmallow/compare/v1.1.3...v1.1.4) - 2022-11-10
```
Up to you but the KeepAChangeLog format doesn't have the extra `[]` around the version number.